frontend/test/app/components/ExperimentCardList.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { assert, expect } from 'chai';
+import { shallow, mount, render } from 'enzyme';
+
+import ExperimentCardList from '../../../src/app/components/ExperimentCardList';
+import ExperimentRowCard from '../../../src/app/components/ExperimentRowCard';
+import Loading from '../../../src/app/components/Loading';
+
+
+const _exp = [
+  { slug: 'foo' },
+  { slug: 'bat' }
+];
+
+describe('ExperimentCardList', () => {
```

Comment:
nit: Also been using a pattern of `app/components/ExperimentCardList` in `describe()` to be more explicit about the test subject